letsdrill.rb

```diff
@@ -1,13 +1,34 @@
 def get_letter_grade(integer)
  
-  #Put your code here!
+  if integer >= 90 then
+    return "A"
+  elsif integer >= 80 then
+    return "B"
+  elsif integer >= 70 then
+    return "C"
+  elsif integer >= 60 then
+    return "D"
+  end
+
+  return "F"
 
 end
 
 def shortest_string(array)
 
-  #Put your code here!
+  if array.length == 0
+    return nil
+  end
+
+  shortest = array[0]
+
+  array.each { |item|
```

Comment:
It's preferred to use the `do/end` syntax for multiline blocks

Reply:
Thanks for all the tips! I swapped out all the styling stuff like you mentioned 👍